src/dragDrop.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars, no-use-before-define */
+export default function dragDrop(tasks) {
```

Comment:
- [ Optional]
As per the learning objective, we have to follow the guidelines of ES6 syntax. Kindly try to use the arrow function.